requirements/edx/base.txt

```diff
@@ -48,9 +48,6 @@ lazy==1.1
 lxml==3.3.6
 mako==0.9.1
 Markdown==2.2.1
---allow-external meliae
```

Comment:
This was an attempt to fix the broken GitHub dependencies locally, but I didn't mean to check it in. I'll remove it.